install-panel.sh

```diff
@@ -909,9 +945,62 @@ perform_install() {
   install_pteroq
   configure_nginx
   [ "$CONFIGURE_LETSENCRYPT" == true ] && letsencrypt
+  [ "$BUILD_PANEL" == true ] && build_panel
   true
 }
 
+choose_branch() {
+  echo -ne "* Would you like to choose a specific version/branch to be installed? (y/N): "
+  read -r CUSTOM_VERSION
+
+  if [[ "$CUSTOM_VERSION" =~ [Yy] ]]; then
+    list_branches
+    update_links
+  fi
+}
+
+list_branches() {
```

Comment:
Not sure about this. I think it should be more generic, so that the user could install anything (for example pull the list from the web or something). We don't want to keep updating the script on every new panel release.

Reply:
Yeah, I thought about that, the thing is, there are a lot of weird branches in the panel, and doing multiple `SED` commands to remove, I don't know if it would be good

Reply:
So, I think that for version you can use tags (as they only have versions there) and then include only 2 branches `v2` and `develop`. `v2` could be deleted after it is released, *Don't forget to add a comment*.

Reply:
Okay, about the versions, I'm not sure if I should include all of them, maybe just keep `1.6.0` and `1.6.6`?